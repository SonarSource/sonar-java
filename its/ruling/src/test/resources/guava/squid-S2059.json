{
'com.google.guava:guava:src/com/google/common/collect/HashBiMap.java':[
526,
],
'com.google.guava:guava:src/com/google/common/reflect/TypeToken.java':[
628,
],
}
