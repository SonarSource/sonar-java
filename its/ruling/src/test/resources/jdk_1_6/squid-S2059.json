{
'jdk_1_6:java/awt/CardLayout.java':[
57,
],
'jdk_1_6:java/awt/Component.java':[
8944,
],
'jdk_1_6:java/awt/MenuComponent.java':[
464,
],
'jdk_1_6:java/awt/ScrollPane.java':[
696,
],
'jdk_1_6:java/beans/beancontext/BeanContextSupport.java':[
300,
],
'jdk_1_6:java/util/TreeMap.java':[
1837,
],
'jdk_1_6:java/util/concurrent/locks/AbstractQueuedLongSynchronizer.java':[
1551,
],
'jdk_1_6:java/util/concurrent/locks/AbstractQueuedSynchronizer.java':[
1776,
],
}
