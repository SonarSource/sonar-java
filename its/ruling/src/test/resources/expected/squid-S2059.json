{
'project:com/sun/corba/se/impl/activation/RepositoryImpl.java':[
405,
444,
],
'project:com/sun/org/apache/xerces/internal/dom/DocumentImpl.java':[
485,
870,
],
'project:com/sun/org/apache/xerces/internal/dom/ParentNode.java':[
1026,
],
'project:java/awt/CardLayout.java':[
74,
],
'project:java/awt/Component.java':[
8993,
],
'project:java/awt/MenuComponent.java':[
475,
],
'project:java/awt/ScrollPane.java':[
717,
],
'project:java/beans/beancontext/BeanContextSupport.java':[
317,
],
'project:java/util/TreeMap.java':[
1859,
],
'project:java/util/concurrent/locks/AbstractQueuedLongSynchronizer.java':[
1611,
],
'project:java/util/concurrent/locks/AbstractQueuedSynchronizer.java':[
1832,
],
'project:javax/sql/rowset/RowSetMetaDataImpl.java':[
944,
],
'project:javax/swing/AbstractButton.java':[
1977,
2321,
],
'project:javax/swing/DefaultCellEditor.java':[
300,
],
'project:javax/swing/ImageIcon.java':[
563,
],
'project:javax/swing/JColorChooser.java':[
727,
],
'project:javax/swing/JComboBox.java':[
1499,
],
'project:javax/swing/JFormattedTextField.java':[
643,
1178,
],
'project:javax/swing/JList.java':[
1809,
],
'project:javax/swing/JMenu.java':[
1114,
1161,
],
'project:javax/swing/JProgressBar.java':[
650,
],
'project:javax/swing/JRootPane.java':[
903,
],
'project:javax/swing/JScrollBar.java':[
704,
],
'project:javax/swing/JSlider.java':[
348,
],
'project:javax/swing/JSpinner.java':[
383,
],
'project:javax/swing/JTabbedPane.java':[
268,
2032,
],
'project:javax/swing/JTextField.java':[
849,
],
'project:javax/swing/JToolBar.java':[
745,
],
'project:javax/swing/JTree.java':[
3362,
],
'project:javax/swing/JViewport.java':[
1276,
],
'project:javax/swing/colorchooser/DefaultSwatchChooserPanel.java':[
212,
221,
],
'project:javax/swing/plaf/basic/BasicComboPopup.java':[
799,
],
'project:javax/swing/text/AbstractDocument.java':[
1764,
],
'project:javax/swing/text/DefaultFormatter.java':[
678,
721,
],
'project:javax/swing/text/DefaultStyledDocument.java':[
1397,
],
'project:javax/swing/text/JTextComponent.java':[
4989,
],
'project:javax/swing/text/StyleContext.java':[
1247,
],
'project:javax/swing/text/StyledEditorKit.java':[
221,
],
}
